Makefile

```diff
@@ -1,49 +1,47 @@
 #
 # github.com/docker/cli
 #
-
 all: binary
 
 # remove build artifacts
 .PHONY: clean
 clean:
-	@rm -rf ./build/* cli/winresources/rsrc_*
+	rm -rf ./build/* cli/winresources/rsrc_*
 
 # run go test
 # the "-tags daemon" part is temporary
 .PHONY: test
 test:
-	@go test -tags daemon -v $(shell go list ./... | grep -v /vendor/)
+	go test -tags daemon -v $(shell go list ./... | grep -v /vendor/)
 
 .PHONY: lint
 lint:
-	@gometalinter --config gometalinter.json ./...
+	gometalinter --config gometalinter.json ./...
 
 .PHONY: binary
 binary:
-	@./scripts/build/binary
+	@echo "WARNING: binary creates a Linux executable. Use cross for macOS or Windows."
+	./scripts/build/binary
 
-# build the CLI for multiple architectures
 .PHONY: cross
 cross:
-	@./scripts/build/cross
+	./scripts/build/cross
 
 .PHONY: dynbinary
 dynbinary:
-	@./scripts/build/dynbinary
+	./scripts/build/dynbinary
 
 .PHONY: watch
 watch:
-	@./scripts/test/watch
+	./scripts/test/watch
 
-# download dependencies (vendor/) listed in vendor.conf
-.PHONY: vendor
+# Check vendor matches vendor.conf
 vendor: vendor.conf
-	@vndr 2> /dev/null
-	@scripts/validate/check-git-diff vendor
+	vndr 2> /dev/null
```

Comment:
I could add `timeout 60` ?

Or I could rearrange this so that it's a phony target for CI called `check-vendor`, and `vendor` can include the output.

WDYT?